test/services/sync/strategies/CSVSyncer/CSVSyncer.unit.test.js

```diff
@@ -75,5 +78,23 @@ describe('CSVSyncer', () => {
 			expect(result[1].class).to.equal('');
 			expect(result[2].class).to.equal('2b');
 		});
+
+		it('should not accept unvalid dates as user birthday', () => {
+			const falsyValues = [false, null, undefined, '', 0]
+			const misfitValues = [Symbol(), [], {}, true]
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Expected Symbol to have a description.](https://app.codacy.com/manual/schul-cloud/schulcloud-server/pullRequest?prid=5220139)